public/app/plugins/panel/pmm-update/UpdatePanel.utils.ts

```diff
@@ -0,0 +1,14 @@
+import { format } from 'date-fns';
+import { enUS } from 'date-fns/locale';
+
+import { ISOTimestamp } from './types';
+
+export const formatDateWithTime = (timestamp: ISOTimestamp) => {
+  const date = new Date(timestamp);
+  return format(new Date(date).valueOf() + date.getTimezoneOffset() * 60 * 1000, 'MMMM dd, H:mm', { locale: enUS });
```

Comment:
Is there a reason why a date is recreated? `date` is already a date object and it should be possible to call `valueOf` directly on it.

Reply:
Good one !